include/bout/paralleltransform.hxx

```diff
@@ -56,6 +56,9 @@ public:
 
   virtual bool canToFromFieldAligned() = 0;
 
+  /// Does a Field3D with ytype require a twist-shift at branch cuts on closed field lines?
+  virtual bool twistShift(bool twist_shift_enabled, YDirectionType ytype) = 0;
```

Comment:
The name should really reflect that this is a predicate, otherwise it sounds like a verb and this will do something. So maybe `requiresTwistShift`?

Also, docstring could be something like:

```cpp
  /// If \p twist_shift_enabled is true, does a `Field3D` with Y direction \p ytype 
  /// require a twist-shift at branch cuts on closed field lines?
  virtual bool twistShift(bool twist_shift_enabled, YDirectionType ytype) = 0;
```